lib/plug/templates/mailbox_viewer/index.html.eex

```diff
@@ -4,203 +4,159 @@
-        z-index: 1000;
-        display: block;
-      }
-
-      .list-group-item:first-child {
-        border-top-left-radius: 0;
-        border-top-right-radius: 0;
-      }
-
-      .list-group-item:last-child {
-        border-bottom-right-radius: 0;
-        border-bottom-left-radius: 0;
-      }
-
-      .list-group-item {
-        border-right: none;
-      }
-
-      .content {
-        padding-right: 0;
-        padding-left: 0;
-      }
-
-      .header {
-        border-bottom: 1px solid #ddd;
-      }
-
-      .header-content {
-        padding: 1rem 2rem;
-      }
-
-      .headers {
-        margin-left: 0;
-        margin-bottom: 0;
-      }
-
-      .headers dt {
-        padding-left: 0;
-      }
-
-      .headers dd {
-        margin-bottom: 0;
-      }
-
-      .body {
-        padding: 1rem 2rem;
-      }
-
-      .body-text {
-        padding: 1.5rem;
-        background-color: #f7f7f9;
-      }
-
-      .body-html {
-        width: 100%;
-        height: 100%;
-        border: 2px solid #f7f7f9;
-      }
-
-      form {
-        margin-bottom: 0;
-      }
-    </style>
+    <script src="https://cdn.tailwindcss.com"></script>
   </head>
   <body>
-    <div class="container-fluid">
-      <div class="row">
-        <div class="col-sm-3 list">
-          <div class="list-group">
-            <%= if Enum.empty?(@emails) do %>
-              <div class="list-group-item">
-                <h6 class="list-group-item-heading">Empty mailbox...</h6>
-              </div>
-            <% else %>
-              <form method="POST" action="<%= to_absolute_url(@conn, "clear") %>">
-                <input name="_csrf_token" type="hidden" value="<%= Plug.CSRFProtection.get_csrf_token() %>">
-                <button class="btn btn-primary btn-block" type="submit">Delete Message Queue</button>
-              </form>
+    <div class="flex">
+      <div class="border-r border-gray-200 h-screen w-96">
+        <div class="flex flex-col h-full overflow-y-auto">
+          <div class="p-4 border-b border-gray-200">
+            <h1 class="text-2xl font-semibold">Mailbox</h1>
+            <p class="text-gray-400 text-sm"><%= length(@emails) %> message(s)</p>
+          </div>
+          <%= unless Enum.empty?(@emails) do %>
+            <div class="flex flex-col h-full">
               <%= for email <- @emails do %>
                 <% id = email.headers["Message-ID"] %>
-                <a href="<%= to_absolute_url(@conn, id) %>" class="list-group-item<%= if @email && @email.headers["Message-ID"] == id, do: " active" %>">
-                  <h6 class="list-group-item-heading"><%= render_recipient(email.from) %></h6>
-                  <p class="list-group-item-text"><%= email.subject %></p>
+                <a href="<%= to_absolute_url(@conn, id) %>" class="flex flex-col border-b border-gray-200 p-4<%= if @email && @email.headers["Message-ID"] == id, do: " bg-gray-100" %>">
+                  <% {name, _email_address} = email.from %>
```

Comment:
![image](https://github.com/swoosh/swoosh/assets/1329716/b48c326c-832a-4757-b935-4b6cee2ef03e)

This would be empty in this case

Reply:
You're right, will fix

Reply:
No, actually render_value with show n/a for the empty string